api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -233,7 +228,26 @@
         <addForeignKeyConstraint constraintName="order_set_attribute_creator_fk" baseTableName="order_set_attribute" baseColumnNames="creator" referencedTableName="users" referencedColumnNames="user_id" />
         <addForeignKeyConstraint constraintName="order_set_attribute_changed_by_fk" baseTableName="order_set_attribute" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id" />
         <addForeignKeyConstraint constraintName="order_set_attribute_voided_by_fk" baseTableName="order_set_attribute" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id" />
-    </changeSet>   
-    
+    </changeSet>
+	
+	<changeSet id="2020-08-29-2200-TRUNK-5821" author="miirochristopher">
+		<preConditions onFail="MARK_RAN" onFailMessage="Index idx_order_number already exists.">
+			<tableExists tableName="orders"/>
+		</preConditions>
+		<comment>Adding index to order_number column in Orders table</comment>
+		<createIndex  indexName="idx_order_number" tableName="orders" unique="true">
```

Comment:
For the index name, can you prefix the table name as done in this file? https://github.com/openmrs/openmrs-core/blob/master/api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/liquibase-schema-only-1.9.x.xml